src/components/d4-module/types.ts

```diff
@@ -1,70 +1,67 @@
 // Divi dependencies.
 import { ModuleEditProps } from '@divi/module-library';
 import {
-  AdminLabelGroupAttr,
-  AnimationGroupAttr,
-  BackgroundGroupAttr,
-  BorderGroupAttr,
-  BoxShadowGroupAttr,
   CssAttr,
-  DisabledOnGroupAttr,
-  FiltersGroupAttr,
-  FontBodyGroupAttr,
-  FontGroupAttr,
-  FormatAttr,
+  FormatBreakpointStateAttr,
   IdClassesGroupAttr,
   InternalAttrs,
   LinkGroupAttr,
-  OverflowGroupAttr,
-  PositionGroupAttr,
-  ScrollGroupAttr,
-  SizingGroupAttr,
-  SpacingGroupAttr,
-  StickyGroupAttr,
   TextGroupAttr,
-  TransformGroupAttr,
-  TransitionGroupAttr,
-  ZIndexGroupAttr,
+  PickedElementAttrs,
+  ModuleMetaAttrs,
+  CssGroupAttr,
 } from '@divi/types';
 
 export interface D4ModuleCssAttr extends CssAttr {
   title?: string;
   content?: string;
 }
 
-export type D4ModuleCssGroupAttr = FormatAttr<D4ModuleCssAttr>;
+export type D4ModuleCssGroupAttr = FormatBreakpointStateAttr<D4ModuleCssAttr>;
 
 export interface D4ModuleAttrs extends InternalAttrs {
+  // CSS options is used across multiple elements inside the module thus it deserves its own top property.
+  css?: CssGroupAttr;
 
-  // Content tab.
-  title?: FormatAttr<string>;
-  content?: FormatAttr<string>;
-  link?: LinkGroupAttr;
-  background?: BackgroundGroupAttr;
-  adminLabel?: AdminLabelGroupAttr;
+  module?: {
+    meta?: ModuleMetaAttrs;
+    advanced?: {
+      link?: LinkGroupAttr;
+      htmlAttributes?: IdClassesGroupAttr;
+      text?: TextGroupAttr;
+    };
+    decoration?: PickedElementAttrs<
+      'animation' |
+      'background' |
+      'border' |
+      'boxShadow' |
+      'disabledOn' |
+      'filters' |
+      'overflow' |
+      'position' |
+      'scroll' |
+      'sizing' |
+      'spacing' |
+      'sticky' |
+      'transform' |
+      'transition' |
+      'zIndex'
+    >;
+  };
+  title?: {
+    innerContent?: FormatBreakpointStateAttr<string>;
+    decoration?: PickedElementAttrs<'font'>;
+  };
+  content?: {
+    innerContent?: FormatBreakpointStateAttr<string>;
+    decoration?: PickedElementAttrs<'font'>;
```

Comment:
It should be `bodyFont`.

Reply:
Addressed in https://github.com/elegantthemes/d5-extension-example-modules/pull/12/commits/ff7169dc17493ee49193e837e2f53adebb5d0ec3